README.md

```diff
@@ -55,6 +55,7 @@ Temporal.configure do |config|
   config.port = 7233
   config.namespace = 'ruby-samples'
   config.task_queue = 'hello-world'
+  config.channel_creds = :this_channel_is_insecure
```

Comment:
nit: Ruby folks are ok with full names, let's probably call this just `credentials` unless you think it might conflict with some other config we'd want to add in the future